src/printer.js

```diff
@@ -1801,14 +1801,20 @@ function printArgumentsList(path, options, print) {
     ]);
   }
 
+  // For next group, there are some exceptions where we don't want
+  // a trailing comma in the last comment.
+  const noTrailingCommaInLastComment = lastArg.type === "ObjectExpression";
```

Comment:
Oh you're right... I was focused on that one https://github.com/jlongster/prettier/issues/538#issue-204423255 but actually this one makes it clear for me now https://github.com/jlongster/prettier/issues/538#issuecomment-276522889.